src/components/SearchResult/SearchResult.tsx

```diff
@@ -0,0 +1,115 @@
+import { noop, truncate } from 'lodash-es';
+import React, { FunctionComponent } from 'react';
+import { Link } from 'react-router-dom';
+
+import { MetaData, MetaDataItem, TagList, Thumbnail, ToggleButton } from '../..';
+import { formatDate, formatDuration } from '../../helpers/formatting';
+
+export interface SearchResultProps {
+	pid: string;
+	type: 'collection' | 'video' | 'audio';
+	thumbnailPath?: string;
+	duration?: number;
+	numberOfItems?: number;
+	description?: string;
+	title: string;
+	link: string;
+	originalCp: string;
+	date: string;
+	tags?: string[];
+	originalCpLink: string;
+	onToggleBookmark?: (id: string, active: boolean) => void;
```

Comment:
id als eerste prop is telkens niet nodig, kan van buitenaf opgevangen worden.
(de andere 3 methods heb je niet nodig na slot changes normaal gezien)